tests/test_aws_credentials_file

```diff
@@ -14,7 +14,7 @@ aws_secret_access_key=yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy
 aws_access_key_id=BBBBBBBBBBBBBBBBBBBB
 aws_secret_access_key=yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy
 
-[bogusprofile]
+[bogus-test-profile]
```

Comment:
the `bogus-test-profile` usage in `TestRunWithoutRegion` is confusing. in the tests, it's deliberately meant to set a profile that is not configured in `tests/test_aws_config_file` to ensure the workflow can fail gracefully. therefore, we don't want to update usages to that everywhere to use `bogus-test-profile`, as we're losing coverage with no gain

feel free to add a code comment in `TestRunWithoutRegion` to call this out, but we should otherwise revert this profile rename

Reply:
Oh yeah, I misunderstood it. I had to update the function to throw an error instead of panic, but `bogusprofile` is back and the snapshot got back to how it was.